src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,120 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, $catalog__card-width);
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      @include square(2);
```

Comment:
The mixin `square` is used here, but ensure that it is defined elsewhere in your SCSS files. If not, this will cause a compilation error.